lib/ts/ink_queue.cc

```diff
@@ -52,6 +52,11 @@
 #include "ts/hugepages.h"
 #include "ts/Diags.h"
 
+
+// in order to compile correctly
```

Comment:
I may be mistaking the line in ink_queue.cc with these.  The ink_queue one may be superfluous now.

```
iocore/eventsystem/ProxyAllocator.cc +29;: :#undef HAVE_LIBJEMALLOC
lib/ts/Arena.cc +28;: :#undef HAVE_LIBJEMALLOC
```